--- WORKSPACE	2018-03-29 12:18:40.000000000 +0800
+++ WORKSPACE	2018-04-18 09:42:06.036181496 +0800
@@ -5,8 +5,7 @@ http_archive(
     sha256 = "6691c58a2cd30a86776dd9bb34898b041e37136f2dc7e24cadaeaf599c95c657",
     strip_prefix = "rules_closure-08039ba8ca59f64248bb3b6ae016460fe9c9914f",
     urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/rules_closure/archive/08039ba8ca59f64248bb3b6ae016460fe9c9914f.tar.gz",
-        "https://github.com/bazelbuild/rules_closure/archive/08039ba8ca59f64248bb3b6ae016460fe9c9914f.tar.gz",  # 2018-01-16
+        "file:///usr/tensorflow-deps/rules_closure.tar.gz"
     ],
 )
 
