--- third_party/jpeg/jpeg.BUILD	2018-03-29 12:18:40.000000000 +0800
+++ third_party/jpeg/jpeg.BUILD	2018-04-18 12:23:40.403741309 +0800
@@ -526,12 +526,12 @@ config_setting(
 
 config_setting(
     name = "armeabi-v7a",
-    values = {"android_cpu": "armeabi-v7a"},
+    values = {"cpu": "armeabi-v7a"},
 )
 
 config_setting(
     name = "arm64-v8a",
-    values = {"android_cpu": "arm64-v8a"},
+    values = {"cpu": "arm64-v8a"},
 )
 
 config_setting(
